src/models/platform.ts

```diff
@@ -58,6 +58,80 @@ export function getRuntimeDisplayName(runtime: Runtime): string {
     }
 }
 
+/**
+ * There is no standard way on Linux to find the distribution name and version.
+ * Recently, systemd has pushed to standardize the os-release file. This has
+ * seen adoption in "recent" versions of all major distributions.
+ * https://www.freedesktop.org/software/systemd/man/os-release.html
+ */
+export class LinuxDistribution {
+    public constructor(
+        public name: string,
+        public version: string,
+        public idLike?: string[]) { }
+
+    public static GetCurrent(): Promise<LinuxDistribution> {
```

Comment:
we should use `camelCasing` naming convention in TypeScript code, so for exampl, `getCurrent` here.  Ideally tslint would flag these.

Reply:
hmm, looks like you just moved this code around, so it's optional if you'd like to update.

Reply:
Correct, this was just moved (it failed tslint use-before-declared rule, since it's used down below on line 208)